SQLite.Net.Tests/FieldSupportTest.cs

```diff
@@ -27,6 +27,16 @@ public void AssertNotSet()
             Assert.That(shouldNotBeSet2, Is.EqualTo(-2));
         }
     }
+
+    public class FieldTestModelWithPkProperty
+    {
+        [PrimaryKey]
+        public int Id { get; set; }
```

Comment:
can you move this to the last declaration position, will be a better unit test

Reply:
Resolved in PR #10